components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/SubscriptionsApi.java

```diff
@@ -82,6 +82,28 @@ public Response subscriptionsSubscriptionIdGet(@ApiParam(value = "Subscription I
     {
     return delegate.subscriptionsSubscriptionIdGet(subscriptionId,accept,ifNoneMatch,ifModifiedSince);
     }
+    @PUT
+    @Path("/{subscriptionId}")
+    
+    
```

Comment:
Why blank lines? Can we avoid the blank links here. There should be a blank link before `@PUT`. Please check code formatting.